Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -1282,8 +1284,6 @@ function Confirm-ServiceConfig{
 
     Confirm-SettingIsNotNull -settingName "common.sqlServerAddress" -settingValue $commonConfig.sqlServerAddress
     Confirm-SettingIsNotNull -settingName "common.metadataDbName" -settingValue $commonConfig.metadataDbName
-    Confirm-SettingIsNotNull -settingName "common.webServerDomain" -settingValue $commonConfig.webServerDomain
-    Confirm-SettingIsNotNull -settingName "common.clientEnvironment" -settingValue $commonConfig.clientEnvironment
```

Comment:
Was clientEnvironment being used for anything? Not seeing it being assigned in anything that was removed, but it looks like it was still being checked for like on this line

Reply:
I could not find any references to it, so I removed it.